app/assets/javascripts/miq_application.js

```diff
@@ -1246,6 +1246,7 @@ function miqInitSelectPicker() {
     style: 'btn-info',
     size: 4
   });
+  $('[title]').not('.selectpicker').tooltip({container: 'none'});
```

Comment:
@fryguy yes this is disabling tooltips on all selects. We don't need tooltips on selects.